In [1]:
from astropy import units as u
from astropy import constants as const 

## Absorbed Dose

The absorbed dose is absorbed energy density. Defined by the quotient of mean energy $\Delta {\bar\epsilon}$ in to mass $\Delta m$ in the limit as the mass approches zero

$$
D:=\lim_{\Delta m\to 0} \frac{\Delta {\bar\epsilon}}{\Delta m}
$$

It's standard unit is gray ($Gy$)

$$
1\,\mathrm{Gy}=\frac{1\,\mathrm{J}}{1\,\mathrm{kg}}
$$

In [2]:
Gy = u.def_unit('Gy',u.J/u.kg)
Gy

Unit("Gy")

A traditional unit for absorbed dose is $rad$

$$
1\,\mathrm{rad} = \frac{100\,\mathrm{erg}}{1\,\mathrm{g}}
$$

In [3]:
rad = u.def_unit('rad',100*u.erg/u.g)
(1*rad).to(Gy)

<Quantity 0.01 Gy>

Thus a $1\,\mathrm{rad}=0.01\,\mathrm{Gy}$

## Kerma

In many circumstances, it is difficult to calculate the absorbed dose from radiation fluence and material properties. 

A closely related deterministic quantity, used only in connection with indirectly ionizing (uncharged) radiation, is the **kerma**, an acronym for **k**inetic **e**nergy of **r**adiation produced per unit **ma**ss. 

$$
K:=\lim_{\Delta m\to 0} \frac{\Delta {\bar{E}_{tr}}}{\Delta m}
$$

$\bar E_{tr}$ is the sum of the initial kinetic energies of all the charged ionizing particles released by interaction of indirectly ionizing particles in matter of mass $m$.

The term "indirectly" means we should consider all the charged particles generated by radiation itself and its uncharged production.

This seems to be due to the technical problem that neutral particles are difficult to detect.

Absorbed dose and kerma are frequently almost equal in magnitude. Under a
condition known as charged-particle equilibrium, they are equal.

To calculate the Kerma of certain energy beam

$$
K=\frac{\mu (E) \Phi}{\rho} \times (E f(E))\\
\dot K = \frac{\mu (E) \phi}{\rho} \times (E f(E))
$$

Here $\mu(E)$ is the linear interaction coefficient, $f(E)$ is the fraction of incident radiation particle's energy that transferred to secondary charged particles and $\Phi$ is he fluence, while $\phi$ is the fluence rate. Thus $\frac{\mu (E) \Phi}{\rho}$ is the number of particles absorbed per unit mass and $E f(E)$ is the averange energy transferred to secondary charged particles per particle.

We can rewrite the calculating equation,
$$
K=\left(\frac{f(E) \mu(E)}{\rho}\right) E \Phi
$$

$f\mu =:\mu_{tr}$ is the "Kerma interaction coefficient".

$$
K(E)=\left(\frac{\mu_{t r}(E)}{\rho}\right) E \Phi
$$


In [16]:
((1*u.cm**2/u.g)*u.MeV*u.cm**-2).to(Gy)

<Quantity 1.60217663e-10 Gy>

Thus if $K$ takes $\mathrm{Gy}$ as unit, $\frac{\mu_{t r}(E)}{\rho}$ takes $\mathrm{cm^2/g}$ as unit, $E$ takes $\mathrm{MeV}$ as unit and $\Phi$ takes $\mathrm{cm^{-2}}$ as unit. The calculation is

$$
K=1.602\times 10^{-10} \left(\frac{\mu_{t r}(E)}{\rho}\right) E \Phi
$$

All the provious process can also be applied to dose $D$.

$$
D(E)=\left(\frac{\mu_{en}(E)}{\rho}\right) E \Phi
$$

> Example 9.1: What are the iron kerma and absorbed dose rates from uncollided
photons $1$ meter from a point isotropic source emitting $10^8$ $
5\mathrm{MeV}$ gamma rays
per second into an infinite water medium?
> $$
\mu_{\mathrm{H_2 O}} = 0.03031\,\mathrm{cm^{−1}}
\\
\left(\frac{\mu_{en}}{\rho}\right)_{\mathrm{Fe}} = 0.01983 \,\mathrm{cm^2/g}
\\
\left(\frac{\mu_{tr}}{\rho}\right)_{\mathrm{Fe}} = 0.02112 \,\mathrm{cm^2/g}
$$

First we need to calculate the flux rate $\phi$

In [27]:
from numpy import pi,exp
Sourse = 10**8 *u.Bq
r = 1 * u.m
mu = 0.03031*u.cm**-1
E = 5*u.MeV

phi = Sourse/(4*pi*r**2)*exp(-mu*r)
phi

<Quantity 384099.34036608 Bq / m2>

Using the formla
$$
K(E)=\left(\frac{\mu_{t r}(E)}{\rho}\right) E \Phi
\\
D(E)=\left(\frac{\mu_{en}(E)}{\rho}\right) E \Phi
$$


In [36]:
mu_tr_mass = 0.02112*u.cm**2/u.g
mu_en_mass = 0.01983*u.cm**2/u.g

D_rate = mu_en_mass*phi*E
D_rate = D_rate.to(Gy/u.s)

K_rate = mu_tr_mass*phi*E
K_rate = K_rate.to(Gy/u.s)

display(D_rate,K_rate)

<Quantity 6.10164131e-10 Gy / s>

<Quantity 6.49857108e-10 Gy / s>

So the answer is
$$
\dot D = 6.1016413\times10^{−10} \,\mathrm{Gy/s}
\\
\dot K = 6.4985711\times10^{−10} \,\mathrm{Gy/s}
$$

## Exposure

**Exposure**, with abbreviation $X$, is used traditionally to specify the radiation field of gamma or $X$-ray photons. It is applied only to photons.

Exposure is defined as the absolute value of the ion charge of one sign produced anywhere in air by the complete stoppage of all negative and positive electrons, except those produced by bremsstrahlung, that are liberated in an incremental volume of air, per unit mass of air in that volume.

To be concise, exposure focus on the amount of charge generated rather than the kinetic energy.

The standard unit of exposure is $\mathrm{C/kg}$. A traditional unit is roentgen,  abbreviated $\mathrm{R}$, $1\,\mathrm{R}=2.58\times10^{-4}\,\mathrm{C/kg}$.

The relaitonship between exposure and krama is the average ionization energy $W$.

$$
K = WX
$$

## Relative Biological Effectiveness

In dealing with the fundamental behavior of biological material or organisms subjected to radiation, radiobiologists define a relative biological effectiveness (RBE) for each type and energy of radiation, and, indeed, for each biological effect or endpoint. 

The RBE is the <u>ratio of</u> the absorbed dose of a reference type of radiation(typically, $250$-kVp x rays or $1.25\,\mathrm{MeV}$ $\mathrm{^{60}Co}$ gamma rays) **producing a certain kind and degree of biological effect** <u>to</u> the absorbed dose of the radiation under consideration required to **produce the same kind and degree of effect**.

RBE is normally determined experimentally and takes into account all factors affecting biological response to radiation in addition to absorbed dose.

## Dose equivalent

The RBE depends on many variables, it is too complicated a concept to be applied in the routine practice of radiation protection or in the establishment of broadly applied standards and regulations.

So in 1964, a group of specialists established **quality factor** $QF$ to define **dose equivalent** $H$

$$
H := QF\times D
$$

$QF$ is often represented as a dimensionless number. However the standard unit of $H$ is sievert($\mathrm{Sv}$), so you can just treat $QF$ with a fixed unit $\mathrm{Sv/Gy}$.

## Effective Dose Equivalent

The Effective Dose Equivalent is calculated using a weighted average of the dose equivalents received by key body organs and tissues. This calculation involves multiplying the absorbed dose (DT) by the tissue weighting factor (wT) and the quality factor (QF), which reflects the type and energy distribution of the radiation. The formula is:

$$
HE = \sum(w_T \cdot QF_T \cdot D_T)
$$

In [ ]:
In dealing with the fundamental behavior of biological material or organisms
subjected to radiation

In [ ]:
a = a.to(Gy)
print(f'{a.value:e} {a.unit}')

8.010883e-04 Gy
